### DISCUSSION TITLE
Is the file format compatible with KeepassXC?

### DISCUSSION THREAD

**Discussion by vonHabsi** [Kunzisoft/KeePassDX#1959] [General]
Is the file format compatible with KeepassXC, according to this description on their KeepassXC site?

> KeePassXC currently uses the KeePass 2.x (.kdbx) password database formats KDBX 3.1 and KDBX 4 as its native file formats.

I want to be able to synchronize the file with my desktop copies automatically.

**Comment by Havokdan** [comment]
Yes, its compatible.
